src/sections/Meshmap/FeaturesSection/useGsapTimeline.js

```diff
@@ -1,23 +1,18 @@
 import { gsap } from "gsap";
 import ScrollTrigger from "gsap/dist/ScrollTrigger";
 import { useEffect, useLayoutEffect } from "react";
-import useHasMounted from "../../../utils/useHasMounted";
 
 gsap.registerPlugin(ScrollTrigger);
 
 const useGsapTimeline = ({ trigger, featureContainerName,yPercent }) => {
 
-  const hasMounted = useHasMounted();
-
   const useIsomorphicLayoutEffect =
-  hasMounted ? useLayoutEffect : useEffect;
-
+  typeof window !== "undefined" ? useLayoutEffect : useEffect;
```

Comment:
Hi @Nikhil-Ladha,

Hope all is well.

There was a console dev warning:

>Warning: React has detected a change in the order of Hooks called by DesignerFeatures. This will lead to bugs and errors if not fixed...

![image](https://github.com/layer5io/layer5/assets/90356410/1dfd5aeb-52df-4f43-afea-c3d9c3f2c667)

that was triggering when the DesignerFeatures. VisualizerFeatures, and CollaboratorFeatures components loaded. I followed it to this hook `useGsapTimeline.js` and noticed that I had earlier replaced the ` typeof window !== "undefined" ` with the `useHasMounted` method. 

After further examination, I realized the [`useHasMounted` method](https://www.joshwcomeau.com/react/the-perils-of-rehydration/#the-solution-9) is meant to replace `typeof window !== "undefined"` to make sure **content** in the "hydrated app matches the original HTML", and that I incorrectly used it in this hook (causing the dev warning).

I reverted to the original implementation `typeof window !== "undefined" ? useLayoutEffect : useEffect;` and the dev warning was cleared.

Thanks!